app/services/api.js

```diff
@@ -30,49 +26,10 @@ export default Service.extend({
   },
 
   request(url, method = 'GET', options = {}) {
-    let endpoint = config.apiEndpoint || '';
-    let token = get(this, 'auth.token');
-
     options.headers = options.headers || {};
     options.headers['Travis-API-Version'] = '3';
-    if (!options.headers['Accept']) {
-      options.headers['Accept'] = 'application/json';
-    }
-
-    if (token) {
-      if (!options.headers['Authorization']) {
-        options.headers['Authorization'] = `token ${token}`;
-      }
-    }
-    options.url = url = `${endpoint}${url}`;
-    options.method = method;
-    options.dataType = options.dataType || 'json';
-    options.contentType = 'application/json';
-
-    if (options.data && options.stringifyData !== false) {
-      options.data = JSON.stringify(options.data);
-    }
-
-    return new EmberPromise((resolve, reject) => {
-      options.error = (jqXHR, textStatus, errorThrown) => {
-        if (get(this, 'features.debugLogging')) {
-          // eslint-disable-next-line
-          console.log(`[ERROR] API responded with an error (${status}): ${JSON.stringify(data)}`);
-        }
-        run(() => {
-          // TODO: in the future we might want to run some handler here
-          // that would process all args
-          reject(jqXHR);
-        });
-      };
-
-      options.success = (payload, textStatus, jqXHR) => {
-        run(() => {
-          resolve(payload);
-        });
-      };
+    options.headers['Accept'] = 'application/json';
```

Comment:
Is this still required? It seems like `ajax` adds it by default, doesn't it? Though, i might be missing something 😅

Reply:
Good point, I think you're right. There were so many things like this where the api and ajax service were doing the same thing, but sometimes in slightly different ways haha. I removed this line and all tests pass and things seem to work, so I think it should be good. Change made :)